validationTemplate/src/ModelCalibrationTemplate/CalibrationTemplate.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Aug  4 09:33:34 2021
+
+@author: GARRM
+"""
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default', DeprecationWarning)
+# standard library
+import os
+import sys
+# external libraries
+# RAVEN libraries
+## caution: using abspath here does not do desirable things on Windows Mingw.
+import _utils as hutils
+raven_path = os.path.abspath(hutils.get_raven_loc()) 
+sys.path.append(raven_path)
+# Importing XML functions from RAVEN
+from utils import xmlUtils
+# Importing Template base class from RAVEN
+from InputTemplates.TemplateBaseClass import Template
+
+
+class CalibrationTemplate(Template):
+    """
+    Template class to create the calibration workflow for the RAVEN validation template
+    """
+    #Template.addNamingTemplates({'step name': '{action}_{subject}', 'distribution': '{var}_dist', 'metric var': '{metric}_{var}'})
+    
+    def __init__(self, distributions, metrics_sync): 
+        """ 
+        Constructor for Calibration Template 
+            @ In, distributions, dict, dictionary of distributions implemented with information
+            @ In, metrics_sync, dict, dictionary of metrics and tags for syncing
+            @ Out, None 
+        """
+        super(CalibrationTemplate, self).__init__()
+        self.distributions = distributions
+        self.metrics_sync = metrics_sync
```

Comment:
HYBRID SQA requires no use of underscores in variable names; it only allows camelBack naming.